README.md

```diff
@@ -1,3 +1,14 @@
 Testing example for begginers
 
+S-Single responsibility principle: This concept say that every class only need must 	will have one job, that’s mean that the system should be separate in 		different modules and everyone should will have a job. That help to make 		easier at the moment when in necessary change the code, because the module 		that need the change is the only that will change.
+
+O-Open/closed principle: This principle mean the classes, modules or functions should 	be “open” by an	 “extension” to allow add code or elements at the classes.
+	Also say that the classes should be closed when other classes have access to 	them, that’s mean that the classes that will called must be are correct and 	don’t have errors. The inheritance it’s based in this principle.
+
+L-Liskov substitution principle: This principle says that an object is a subtype of 	other object, in the code the first object can be replaced by elements of the 	second element and the change must’n create problems in the source code.
+	This principle is useful to create and design the hierarchies of the classes.
+
+I-Interface segregation principle: The principle states that the users not be forced 	to use functions or classes that they not needed. To make this possible is 		necessary divided the jobs in “role interface” and the user only use the role 	that’s need. Because if all job are in one class and is necessary make a 		change, this may generate many errors in the source code.
+
```

Comment:
that the users won't be forced. classes that they don't need.  To make this possible is necessary to divide. the user only use the role 	that it needs.  Because if all job are in one class and is necessary to make a 		change.  

Please fix all of the comments and commit them.